### DISCUSSION TITLE
init.lua configuration for Neovide?

### DISCUSSION THREAD

**Discussion by indicozy** [neovide/neovide#1722] [Q&A]
Hi there!

Even if neovide was made for neovim which is using lua for configuration by default, I could not find how to customize Neovide in lua. (Configuration page)[https://neovide.dev/configuration.html] has shown how to configure neovide in VimScript but not in Lua. How can I configure neovide in Lua?

**Comment by jx11r** [comment]
```lua
if vim.g.neovide then
  vim.opt.guifont = 'FiraCode Nerd Font:h10'
  vim.g.neovide_hide_mouse_when_typing = true
  -- etc
end
```
